public/docs/ts/latest/guide/style-guide.jade

```diff
@@ -1870,6 +1871,280 @@ a(href="#toc") Back to top
 
 +makeExample('style-guide/ts/09-01/app/heroes/shared/hero-button/hero-button.component.ts', 'example', 'app/heroes/shared/hero-button/hero-button.component.ts')
 :marked
+  
+a(href="#toc") Back to top
+
+.l-main-section
+:marked
+  ## Accessibility
+
+  Make your applications accessible to all forms of navigation, including keyboard-only navigation and assistive technologies.  
+
+  ### <a id="10-01"></a>Provide your page language
+  #### <a href="#01-01">Style 10-01</a>
+.s-rule.do
+  :marked
+    **Do** provide a `lang` attribute on your `html` tag.
+
+.s-why
+  :marked
+    **Why?** Assistive technologies, such as screen readers, need to know the language of the page.
+    
+.s-why
+  :marked
+    **Why?** Language tools, such as spell checkers and translators adapt to the given language.
+
+.s-why.s-why-last
+  :marked
+    **Why?** This enables styling and font selection based on the page language.    
+
+:marked
++makeExample('style-guide/ts/10-01/app/index.avoid.html', 'page-lang', 'index.html')(avoid=1)
+
+:marked
+  Provide the ISO language code as well the country code (where appropriate).
```

Comment:
We are not putting messages between avoid and do (correct me if I am wrong). We could put that as a **Consider**